### PR TITLE
Update BUG.md

### PR DESCRIPTION
**Pull request by Hixie**
Let's try simplifying the template to see if people are able to give us the information we actually need.

Things like target platform should be in the logs so if they give us that we don't need to ask for it specifically. People seem to generally be good about saying "iOS only" etc. They don't seem to be good about giving us repro test cases.

cc @timsneath

### CODE REVIEW

**Review by dnfield — APPROVED** [review]
LGTM

File: .github/ISSUE_TEMPLATE/BUG.md [hunk 1]
```diff
@@ -23,32 +23,18 @@ assignees: ''
...
-     Please tell us which target platform(s) the problem occurs (Android / iOS / Web / macOS / Linux / Windows)
-     Which target OS version, for Web, browser, is the test system running?
-     Does the problem occur on emulator/simulator as well as on physical devices?
--->
+**Expected results:** <!-- what did you see? -->
 
-**Target Platform:**
-**Target OS version/browser:**
-**Devices:**
+**Actual results:** <!-- what did you want to see? -->
```
Comment (Zazo032) [inline .github/ISSUE_TEMPLATE/BUG.md]: Maybe this comment is for **Expected results:**?

Comment (timsneath) [reply .github/ISSUE_TEMPLATE/BUG.md]: Good catch -- yeah, looks like this is flipped around.

**Comment by Hixie** [comment]
updated, thanks @Zazo032

**Pull request merged**